src/tests/resources/modeler/epic2/us2.5/study_2.5.4/parameters.yml

```diff
@@ -0,0 +1,6 @@
+solver: sirius
```

Comment:
sirius doesn't support MILP problems

Reply:
thanks I forgot. maybe we should add an explicit exception / log